projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -100,18 +99,6 @@ def __prepare_vdk_run_command(arguments, path):
             cmd.append(f"{arguments}")
         return cmd
 
-    @staticmethod
-    def delete_job(name: str, team: str) -> str:
-        """
-        Execute `delete job`.
-        :param name: the name of the data job that will be deleted
-        :param team: the team of the data job that will be deleted
-        :return: message that the job is deleted
-        """
-        cmd = JobDelete(RestApiUrlConfiguration.get_rest_api_url())
-        cmd.delete_job(name, team)
-        return f"Deleted the job with name {name} from {team} team. "
-
```

Comment:
why are we deleting this?

Reply:
@gageorgiev we actually removed the delete option from the UI, and I forgot to remove it from here 
The reason we decided to remove delete from Jupyter UI is that it is kind of "dangerous" operation and it is better for people to remov jobs from the Operations UI or using the cli